### PR TITLE
Feature/rel 3222

### PR DESCRIPTION
**Pull request by leashma**
Sebastian showed me the proper changes to make in ITCrequest.scala and ConfigExtractor.scala. Then all I did was remove the selection in GSAOI ITC webform. This pull request also includes changes made related to REL-503. Had one unmerged pull request open before making changes for this one.

### CODE REVIEW

File: bundle/edu.gemini.itc.web/src/main/scala/edu/gemini/itc/web/ITCRequest.scala [hunk 1]
```diff
@@ -185,8 +187,11 @@ object ITCRequest {
...
     val filter      = r.enumParameter(classOf[Gsaoi.Filter])
     val readMode    = r.enumParameter(classOf[Gsaoi.ReadMode])
-    val gems        = gemsParameters(r)
-    GsaoiParameters(filter, readMode, gems)
+    val iq          = obsConditionParameters(r).iq
+    ConfigExtractor.createGsaoiParameters(filter, readMode, iq) match {
+      case \/-(p) => p
+      case -\/(t) => throw new IllegalArgumentException(t)
+    }
   }
```
Comment (sraaphorst) [inline bundle/edu.gemini.itc.web/src/main/scala/edu/gemini/itc/web/ITCRequest.scala]: You can also delete `gemsParameters` from this class as it isn't being used anymore.

I should have pointed that out, but I guess I completely forgot.

Comment (leashma) [reply bundle/edu.gemini.itc.web/src/main/scala/edu/gemini/itc/web/ITCRequest.scala]: It is deleted in my local repository, tried to push newest changes, but git tells me there's nothing new to commit.

Comment (sraaphorst) [reply bundle/edu.gemini.itc.web/src/main/scala/edu/gemini/itc/web/ITCRequest.scala]: Did you do a `commit -am "REL-3222: Removed gemsParameters"` first?

I'm looking at the file in this commit and I still see:
```
def gemsParameters(r: ITCRequest): GemsParameters = {
    val avgStrehl  = r.doubleParameter("avgStrehl") / 100.0
    val strehlBand = r.parameter("strehlBand")
    GemsParameters(avgStrehl, strehlBand)
}
```

in `ITCRequest.scala`.

Comment (leashma) [reply bundle/edu.gemini.itc.web/src/main/scala/edu/gemini/itc/web/ITCRequest.scala]: Just re-committed changes now.

**Pull request merged**